src/Model/Behavior/RegisterBehavior.php

```diff
@@ -197,4 +197,22 @@ public function getRegisterValidators($options)
 
         return $validator;
     }
+
+    /**
+     * @param EntityInterface $user User information
+     * @param array $options ['tokenExpiration]
+     * @return bool|EntityInterface
+     */
+    public function resendValidationEmail($user, $options)
+    {
+        $tokenExpiration = Hash::get($options, 'token_expiration');
```

Comment:
I would check if the user is already active first, to prevent sending an activation emails to active users